.buildkite/pipeline.yml

```diff
@@ -56,4 +56,18 @@ steps:
             - "CI_REDSHIFT_DBT_PASS"
             - "CI_REDSHIFT_DBT_USER"
     commands: |
-      bash .buildkite/scripts/run_models.sh redshift
\ No newline at end of file
+      bash .buildkite/scripts/run_models.sh redshift
+
+  - label: ":databricks: Run Tests - Databricks"
```

Comment:
```suggestion
  - label: ":bricks: Run Tests - Databricks"
```

Don't think there's a :databricks: emoji =P

Reply:
isn't it this? 
<img width="209" alt="image" src="https://user-images.githubusercontent.com/65564846/216192883-33174240-23a2-4142-9191-288c7558bea9.png">

Reply:
Will it show up in Github though? Typing :databricks: here just leaves :databricks:

Reply:
ignore, this is for Buildkite, where it works fine =P